docs/tutorials/vagrant-api.rst

```diff
@@ -0,0 +1,189 @@
+
+Post a workflow to a specific node by running the following command.
+
+In the following example, to post a workflow to Reset a Note, the Node_id is obtained by the "curl localhost:8080/api/2.0/nodes | jq '.' "API.
+
+.. code::
+ 
+   vagrant@rackhd:~$ curl -X POST -H 'Content-Type: application/json' 127.0.0.1:8080/api/2.0/nodes/<Node_id>/workflows?name=Graph.Reset.Node | jq '.'
+
+Then the vNode is powered cycle and rebooted.
+
+.. image:: ../_static/issue_workflow.png
+     :align: center
+
+**Redfish API - Chassis**
+
+List the Chassis that is managed by RackHD (equivalent to the enclosure node in REST API 2.0), by running the following command.
+
+.. code::
+
+  vagrant@rackhd:~$ curl 127.0.0.1:8080/redfish/v1/Chassis| jq '.'
+
+
+.. image:: ../_static/redfish_chasis.png
+     :align: center
+
+**Redfish API - System**
+
+1. In the rackhd-server, list the System that is managed by RackHD (equivalent to compute node in API 2.0), by running the following command
+
+.. code::
+  
+ vagrant@rackhd:~$ curl 127.0.0.1:8080/redfish/v1/Systems| jq '.'
+
+2. Use the mouse to select the **System-ID** as below example, then the ID will be in your clipboard. This ID will be used in the following steps.
+
+
+.. image:: ../_static/redfish_sys.png
+     :align: center
+
+**Redfish API - SEL Log**
+
+.. code::
+
+   vagrant@rackhd:~$ curl 127.0.0.1:8080/redfish/v1/systems/<System-ID>/LogServices/Sel| jq '.'
+
+.. image:: ../_static/redfish_sel.png
+     :align: center
+
+**Redfish API - CPU info**
+
+.. code::
+
+  vagrant@rackhd:~$ curl 127.0.0.1:8080/redfish/v1/Systems/<System-ID>/Processors/0| jq '.'
+
+.. image:: ../_static/redfish_cpu.png
+     :align: center
+
+**Redfish API - Helper**
+
+Show the list of RackHD Redfish APIs' by running below command:
+
+.. code::
+  
+  vagrant@rackhd:~$ curl 127.0.0.1:8080/redfish/v1| jq '.'
+
+.. image:: ../_static/redfish_helper.png
+     :align: center
+
+Swagger UI
+-----------
+
+The following steps provide an overview of how to use the Swagger UI.
+
+1. Open "Chrome" and then type ``http://<IP>:9090/swagger-ui`` in the URL address box. The <IP> should be set to the node ip where the environment is set up. For example, if you set up environemt on windows, the <IP> should be set to ``localhost``. If you set up environment on Linux, you can get node ip first by the command ``ifconfig eth0`` and then replace <IP>.
+
+2. You can ignore the Update Keys button and Login button. They take no effect when "authEnabled == false" is set in the RackHD configuration file (/opt/monorail/config.json).
+
+3. Click /api/2.0 > /redfish/v1, to expand the API list.
+
+.. image:: ../_static/swagger-ui.png
+     :align: center
+
+4. Get a Node ID by scrolling down the /API 2.0 list and clicking the "/nodes" API.
+
+5. Click Get a list of nodes on the right side of the bar, to expand the details for this API.
+
+The details for this API are displayed (for example, description, parameters, response code).
```

Comment:
This line can be moved to the end of the last paragraph, otherwise it shows like this
```
    5. Click Get a list of nodes on the right side of the bar, to expand the details for this API.

The details for this API are displayed (for example, description, parameters, response code).

    6. Click Try it out! button, to invoke this API.
```